tardis/utilities/utils.py

```diff
@@ -24,11 +24,47 @@ def htcondor_cmd_option_formatter(options: AttributeDict) -> str:
     return cmd_option_formatter(options, prefix="-", separator=" ")
 
 
+def htcondor_status_cmd_composer(
+    attributes: AttributeDict,
+    options: Optional[AttributeDict] = None,
+    constraint: Optional[str] = None,
+) -> str:
+    """
+    Composes an `condor_status` command string from attributes (classads), options,
+    and an optional constraint. This function does not execute the command,
+    it only returns the assembled command string.
+
+    :param attributes: Mapping of attribute names to values, used to construct
+           the `-af:t` argument.
+    :type attributes: AttributeDict
```

Comment:
General feedback: I recommend to avoid `:type …:` docstrings if you already have type hints. The various inspection and documentation tools have no problem picking up the type hints. Adding the type again in the docstring just duplicates the information in a less accessible way.

Reply:
Thanks! I have removed from the files I touched, for everything else I have create #381.